README.md

```diff
@@ -16,77 +25,87 @@ How to contribute?
+
+### Active
 
 - https://github.com/Cisco
+- https://github.com/Cisco-system-traffic-generator
+- https://github.com/Cisco-Talos
 - https://github.com/CiscoCloud
+- https://github.com/CiscoCXSecurity
 - https://github.com/CiscoDevNet
 - https://github.com/CiscoIOx
-- https://github.com/ciscospark
-- https://github.com/datacenter
 - https://github.com/CiscoSystems
-- https://github.com/cisco-open-source
-- https://github.com/CiscoVE
-- https://github.com/CiscoDevNet
-- https://github.com/CiscoUKIDCDev
-- https://github.com/Cisco-USTC
-- https://github.com/CiscoZeus
-- https://github.com/CiscoKorea
-- https://github.com/CIscoTrainingonDemand
 - https://github.com/CiscoUcs
-- https://github.com/Cisco-oss-eng
-- https://github.com/Cisco-system-traffic-generator
-- https://github.com/CiscoVIRL
+- https://github.com/csap-platform
+- https://github.com/datacenter
+- https://github.com/webex
+
+### Not recently updated
+
+- https://github.com/Cisco-cloud
+- https://github.com/Cisco-Meraki-CFG
+- https://github.com/Cisco-nam
 - https://github.com/Cisco-OCPChina
+- https://github.com/cisco-open-source
+- https://github.com/Cisco-oss-eng
+- https://github.com/Cisco-USTC
 - https://github.com/Ciscocommerce
-- https://github.com/jungo-home
-- https://github.com/CiscoSystems-Netconf
-- https://github.com/Cisco-Meraki-CFG
-- https://github.com/CiscoPackagingExpress
-- https://github.com/CiscoHackathon
 - https://github.com/CiscoCTA
-- https://github.com/Cisco-nam
-- https://github.com/Ciscosspg
-- https://github.com/Cisco-cloud
-- https://github.com/LearningAtCisco
+- https://github.com/CiscoHackathon
+- https://github.com/CiscoKorea
+- https://github.com/CiscoPackagingExpress
+- https://github.com/CiscoSystems-Netconf
+- https://github.com/CiscoUKIDCDev
+- https://github.com/CiscoVE
+- https://github.com/CiscoVIRL
+- https://github.com/CiscoZeus
+
+### Archived
+
+- https://github.com/ciscospark
+- https://github.com/portcullislabs
+
+### No public repositories
+
 - https://github.com/Cisco-china
-- https://github.com/Ciscosystem
-- https://github.com/CiscoDevSeries
+- https://github.com/Cisco-DCG-Power
 - https://github.com/Cisco-TestScheduling
-- https://github.com/CISCO9
 - https://github.com/Cisco001
-- https://github.com/Cisco-DCG-Power
-- https://github.com/csap-platform
-- https://github.com/portcullislabs
-- https://github.com/CiscoCXSecurity
+- https://github.com/CISCO9
+- https://github.com/CiscoDevSeries
+- https://github.com/Ciscosystem
 - https://github.com/CiscoSystemsInc
-- https://github.com/vrtadmin
-- https://github.com/Cisco-Talos
+- https://github.com/CiscoTrainingonDemand
+- https://github.com/LearningAtCisco
+
+### Private/deleted/no longer exists
+
+- https://github.com/Ciscosspg
+- https://github.com/jungo-home
 
-Development
-===========
+## Development
 
-* Tested with Node v5.3.0
-* Fork/Clone this repo
-* `npm install`
-* `gulp`
+- Tested with Node v5.3.0
+- Fork/Clone this repo
+- `npm install`
+- `gulp`
 
```

Comment:
I'll go ahead and merge, but these instructions don't work for me locally -- @fluffy how is the site built locally? Node version is up around v16 now.

Reply:
Zero idea - I always thought this site was sort of a bad idea because it was not clear who would keep it fresh in the long term. But really happy to see people cleaning it up.